### DISCUSSION TITLE
[slick-window-arrangement] Also snap on resizing and inner edges

### DISCUSSION THREAD

**Discussion by ygoe** [ramensoftware/windhawk-mods#264] [General]
I found this mod today and it immediately reminded me of the old days with a little tool called **allSnap**. Somebody still knows that one? I'm going to try it out for a while now and see if it's still as useful as back then.

Just one thing: It snaps windows while moving them, but not while resizing. So I can align windows, but not make them to fill up the space. (Attention: This might interfere with existing screen edge snap effects from Windows itself.) And it also only aligns at outer edges, not at inner edges. So it's impossible to place two windows exactly next to each other, one will often be a bit further up or down than the other. Maybe a few more options would be good for this behaviour.

Interestingly, windows won't snap on the Windhawk window. Is that intentional?

**Comment by m417z** [comment]
> It snaps windows while moving them, but not while resizing.

> And it also only aligns at outer edges, not at inner edges.

These are limitations I'm aware of. I might get back to it one day and fill the gaps.

> Interestingly, windows won't snap on the Windhawk window. Is that intentional?

No, and they snap on my computer. Not sure why they don't for you.

**Comment by krwudtke** [comment]
This is a great mod. Thank you for all of your work on this and others.
It would be even better to have windows snap when resizing them, too.

**Comment by adi-works** [comment]
Can you also add vertical snapping?
